packages/web-awesome/src/components/MetadataButton/index.tsx

```diff
@@ -19,7 +19,7 @@ export const MetadataButton: FunctionalComponent<MetadataButtonProps> = ({
   ...rest
 }) => {
   return (
-    <button
+    <div
```

Comment:
I don't like an idea to use `div` as an interactive elements. Let's keep using `button` here